src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -233,6 +278,43 @@
     text: az backup protection enable-for-azurefileshare --policy-name MyPolicy --resource-group MyResourceGroup --vault-name MyVault --storage-account MyStorageAccount --azure-file-share MyAzureFileShare
 """
 
+helps['backup protection enable-for-azurewl'] = """
+type: command
+short-summary: Start protecting a previously unprotected workload within an Azure VM as per the specified policy to a Recovery services vault. Provide the workload details as a protectable item.
+examples:
+  - name: Start protecting a previously unprotected workload within an Azure VM as per the specified policy to a Recovery services vault. Provide the workload details as a protectable item.
+    text: az backup protection enable-for-azurewl --policy-name MyPolicy --resource-group MyResourceGroup --vault-name MyVault --protectable-item-name ItemName --protectable-item-type SQLInstance --server-name Myserver --workload-type MSSQL
+"""
+
+helps['backup protection auto-enable-for-azurewl'] = """
+type: command
+short-summary: Automatically protect all existing unprotected DBs and any DB which will be added later with the given policy. Azure backup service will regularly scan auto-protected containers for any new DBs and automatically protect them.
+examples:
+  - name: This commands allows users to automatically protect all existing unprotected DBs and any DB which will be added later with the given policy.  Azure backup service will then regularly scan auto-protected containers for any new DBs and automatically protect them.
+    text: az backup protection auto-enable-for-azurewl --policy-name MyPolicy --resource-group MyResourceGroup --vault-name MyVault  --protectable-item-name ItemName --protectable-item-type SQLInstance --server-name Myserver --workload-type MSSQL
+"""
+
+helps['backup protection auto-disable-for-azurewl'] = """
+type: command
+short-summary: Disables auto-protection for the specified item.
+examples:
+  - name: Disables auto-protection for the specified item.
+    text: az backup protection auto-disable-for-azurewl --resource-group MyResourceGroup --vault-name MyVault --item-name MyItemName
+"""
+
+helps['backup recoveryconfig'] = """
+type: group
+short-summary: Recovery configuration of a backed up item.
+"""
+
+helps['backup recoveryconfig show'] = """
+type: command
+short-summary: This command constructs the recovery configuration of a backed up item such as SQL DB.
```

Comment:
The same.

Reply:
@sambitratha : "Construct the recovery configuration of an Azure workload backed up item"